web/src/engine/dom-utils/src/stylesheets.ts

```diff
@@ -144,42 +162,55 @@ export class StylesheetManager {
         // with embedded ttf or woff.  svg mostly works so is a better initial
         // choice on the Android browser.
         if(svg != '') {
-          s0.push("url('"+svg+"') format('svg')");
+          source = "url('"+svg+"') format('svg')";
         }
```

Comment:
SVG is dead for fonts. We shouldn't be perpetuating if we are changing anyway.

l.161 references Android 4.2, 4.3 which we no longer support. We need to clean that up too.

And I see a reference to EOT also. Even deaderer